### ISSUE TITLE
[Feature] : Subscription Number Cruncher Calculator 

### ISSUE THREAD

**Issue by aasthakourav20** [Rakesh9100/CalcDiverse#1337]
### Description

Hey , I would like to add Subscription Number Cruncher Calculator , that calculates the cost of plans for different subscription types and durations.
Kindly assign this task to me under GSSoC'24

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a GSSOC'24 contributor.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by jessicachawla** [comment]
elaborate please; too vague.
will you be taking just one subscription into account or multiple? can the costs be different if you are taking multiple subscriptions into account? can the durations be different for each?

**Comment by Rakesh9100** [comment]
Yes, what is this calculator about? Pls elaborate properly and if you have any supporting screenshots @aasthakourav20
